### ISSUE TITLE
how to make it to ignore certain window (zorin-menu)?

### ISSUE THREAD

**Issue by ahmeteid7** [felixfung/skippy-xd#57]
I'm using zorin lite xfce , i tried skippy-xd on it , it's good, but i notice it deals with zorin-lite-menu as a minimize window and show the absence of it.

**Comment by felixfung** [comment]
Sorry what do you mean by "show the absence of it"?

**Comment by ahmeteid7** [comment]
> Sorry what do you mean by "show the absence of it"?

![image](https://github.com/felixfung/skippy-xd/assets/115186023/93e908cc-366e-4141-a7f6-10897c85c7b9)

i mean when minimizing it,, as u can see the first window (left one)

**Comment by felixfung** [comment]
Can you please share the xprop output for the menu window?

**Comment by ahmeteid7** [comment]
> Can you please share the xprop output for the menu window?

xprop
_NET_WM_ALLOWED_ACTIONS(ATOM) = _NET_WM_ACTION_CLOSE, _NET_WM_ACTION_ABOVE, _NET_WM_ACTION_BELOW, _NET_WM_ACTION_FULLSCREEN, _NET_WM_ACTION_MOVE, _NET_WM_ACTION_RESIZE, _NET_WM_ACTION_MAXIMIZE_HORZ, _NET_WM_ACTION_MAXIMIZE_VERT, _NET_WM_ACTION_CHANGE_DESKTOP, _NET_WM_ACTION_STICK
_NET_WM_DESKTOP(CARDINAL) = 4294967295
_NET_WM_STATE(ATOM) = _NET_WM_STATE_STICKY, _NET_WM_STATE_MODAL, _NET_WM_STATE_SKIP_PAGER, _NET_WM_STATE_SKIP_TASKBAR, _NET_WM_STATE_ABOVE, _NET_WM_STATE_FOCUSED
_NET_FRAME_EXTENTS(CARDINAL) = 0, 0, 0, 0
WM_STATE(WM_STATE):
		window state: Normal
		icon window: 0x7b873c00
WM_HINTS(WM_HINTS):
		Client accepts input or input focus: True
		Initial state is Normal State.
		window id # of group leader: 0x1800001
_GTK_THEME_VARIANT(UTF8_STRING) = "dark"
XdndAware(ATOM) = BITMAP
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x2, 0x0, 0x0, 0x0, 0x0
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_POPUP_MENU
_NET_WM_SYNC_REQUEST_COUNTER(CARDINAL) = 25165996, 25165997
_NET_WM_USER_TIME_WINDOW(WINDOW): window id # 0x18000ab
WM_CLIENT_LEADER(WINDOW): window id # 0x1800001
_NET_WM_PID(CARDINAL) = 4974
WM_LOCALE_NAME(STRING) = "en_US.UTF-8"
WM_CLIENT_MACHINE(STRING) = "eid-HP-Compaq-6005-Pro-SFF-PC"
WM_NORMAL_HINTS(WM_SIZE_HINTS):
		program specified location: 0, 0
		program specified minimum size: 325 by 434
		program specified base size: 0 by 0
		window gravity: NorthWest
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
WM_CLASS(STRING) = "wrapper-2.0", "Wrapper-2.0"
WM_ICON_NAME(STRING) = "Zorin Menu Lite"
_NET_WM_ICON_NAME(UTF8_STRING) = "Zorin Menu Lite"
WM_NAME(STRING) = "Zorin Menu Lite"
_NET_WM_NAME(UTF8_STRING) = "Zorin Menu Lite"

**Comment by felixfung** [comment]
Hmm I see `_NET_WM_STATE_SKIP_PAGER`, as well as `_NET_WM_STATE_SKIP_TASKBAR`, which should have done the job...

Can you please try changing this property:
```
 _NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_POPUP_MENU
```

to something like `_NET_WM_WINDOW_TYPE_DOCK`?

Sorry for super late reply, been completely overwhelmed in other tasks in life...

**Comment by ahmeteid7** [comment]
thanks that solved it. i changed it to _NET_WM_WINDOW_TYPE_DOCK
`xprop -id 0x6600091 -f _NET_WM_WINDOW_TYPE 32a -set _NET_WM_WINDOW_TYPE _NET_WM_WINDOW_TYPE_DOCK`

i notice that happen also with the apps that work in background which has icon in systray like stacer and download manager apps, also the gnome software store,
i will try that solution to them too.

**Comment by felixfung** [comment]
Thanks, will put in the code to make it work for `_NET_WM_WINDOW_TYPE_POPUP_MENU` also. But been super busy, and probably will not be able to do it any time soon.

**Comment by felixfung** [comment]
Hi @ahmeteid7 can you please try https://github.com/felixfung/skippy-xd/pull/59

**Comment by felixfung** [comment]
Closed by #59
